app/src/main/java/com/infomaniak/mail/ui/main/search/SearchFragment.kt

```diff
@@ -98,6 +98,7 @@ class SearchFragment : TwoPaneFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        requireActivity().window.statusBarColor = requireContext().getColor(R.color.backgroundColor)
```

Comment:
Why did this need to be added?

Reply:
Oversight when decoupling TwoPanelFragment from its children.